test/test_tensor_creation_ops.py

```diff
@@ -3170,6 +3171,58 @@ def test_tensor_from_non_writable_numpy(self, device):
             self.assertEqual(len(w), 0)
 
 
+class TestTensorCreationMeta(TestCase):
```

Comment:
@bdhirsh any feedback on my last comment?

Reply:
Hey, sorry for the delay.

> internally calls torch.isnan which does not support the meta backend. So assertEqual itself raises NotImplementedError and the whole test gets "swallowed" before any assertions could take place. 

Oh okay, so most of the existing non-meta tests don't really test meta tensors well, because of this?

I think two reasonable options are problably either:

(1) Just get `isnan` to work properly with meta tensors, so you can re-use those tests

(2) Keep the meta-specific `TestTensorCreationMeta` tests you have around for now, but with a big comment that says they should be fully killed once `isnan` works with meta tensors.

I think (1) would be better :) But I totally get it if you don't want to add scope to what you're working on